JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
-DevExtreme Data Editors provide a&nbsp;powerful and seamless way to&nbsp;perform data validation. In&nbsp;this demo, editors are collected in&nbsp;an&nbsp;HTML&nbsp;form. Each editor is&nbsp;accompanied by&nbsp;the Validator component validating the editor against a&nbsp;set of&nbsp;specified rules. Editors are validated each time their value is&nbsp;changed. All validation errors are displayed in&nbsp;the ValidationSummary component.
+This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on a page with enabled data validation. In this demo, the editors are grouped in an HTML form.
 
+To enable data validation for an editor, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach multiple validation rules to one component. The following list contains all available validation rule types:
 
+- [required](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)    
+A validation rule that demands that the validated field has a value.
 
-In&nbsp;the ASP.NET MVC&nbsp;demo, editors extract validation rules from data annotations attributed to&nbsp;the fields of&nbsp;the model (see the _EditorsViewModel.cs_ file).
+- [email](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/)    
+A validation rule that demands that the validated field matches the Email pattern.
 
+- [async](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/)    
+A custom validation rule used for server-side validation. Implement the [validationCallback](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/#validationCallback) function to validate the target value.
 
+- [compare](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/)    
+A validation rule that demands that the validated editor's value is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares to the validated value.
```

Comment:
```suggestion
A validation rule that requires the validated editor's value to equal 
the value of the specified expression. To apply this rule, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value against which this component compares the validated value.
```
Your paragraph didn't make sense to me, so I rewrote it. However, since I couldn't really understand what you wanted to say, we need to make sure that the technical meaning is correct in my rewrite.

Reply:
Yes, everything is correct, I guess I got confused in the last part of the sentence